clients/hmsbridge/core/pom.xml

```diff
@@ -131,6 +131,10 @@
           <groupId>commons-logging</groupId>
           <artifactId>commons-logging</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.glassfish</groupId>
```

Comment:
This pom change and the one above are weird.

Reply:
the one above is because we put the spark3 dependency w/ exclusions into the parent pom. This one is already on master, not sure why its showing: https://github.com/projectnessie/nessie/pull/848/files